serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -531,7 +531,91 @@ kogito.sw.functions.greet.timeout=5000 <1>
 ----
 <1> Time in milliseconds
 
-== Custom function types
+== Rest Custom Function 
+
+Serverless Workflow Specification defines the xref:service-orchestration/orchestration-of-openapi-based-services.adoc[OpenAPI function type], which is the preferred way to interact with existing REST servers. 
+However, sometimes a workflow shoud interact with several REST APIs which are not described using an OpenAPI specification file. Since generating such file for these services might be a tedious exercise, {product_name} offers REST custom type as a shorcut. 
```

Comment:
```suggestion
However, sometimes a workflow should interact with several REST APIs that are not described using an OpenAPI specification file. Since generating such files for these services might be tedious, {product_name} offers REST custom type as a shortcut. 
```